actors.md

```diff
@@ -688,68 +675,36 @@ func SubmitPost(proofs PoStProof, faults []FaultSet, recovered Bitfield, done Bi
 		TransferFunds(msg.From, msg.Value-feesRequired)
 	}
 
-	if !CheckPostProof(miner.SectorSize, proof, faults) {
+    seed := GetRandFromBlock(self.ProvingPeriodStart + ProvingPeriodDuration(self.SectorSize) - POST_CHALLENGE_TIME)
```

Comment:
If this is the `GetRandFromBlock` defined below in the mining.md doc, then this is incorrect. the randomness needs to come from exactly the block at `ProvingPeriodEnd - POST_CHALLENGE_TIME`. Any lookback will give attackers an advantage and degrade the security argument that RationalPoSt makes.

Reply:
BLOCKING ^

Reply:
@sternhenri and I thought actually that the lookback was good to avoid another issue he was concerned about.

Reply:
also how does the lookback give attackers an advantage? could you explain?

Reply:
I have thought about this some more and you are right, we need to take the challenge directly from this block, as otherwise it can be precomputed in advance.

Reply:
fixed